sdk/src/kalix.ts

```diff
@@ -237,17 +243,34 @@ export interface Component {
    * @internal
    */
   register(allComponents: ServiceMap): ComponentServices;
+
+  /**
+   * Internal method for setting up the component.
+   */
+  preStart(settings: PreStartSettings): void;
```

Comment:
Add an `@internal` annotation to mark properly as internal, and it will also be excluded from the packaged type declarations.

```suggestion
  /**
   * Internal method for setting up the component.
   *
   * @internal
   */
  preStart(settings: PreStartSettings): void;
```